packages/models-library/tests/test_models_fit_schemas.py

```diff
@@ -22,6 +21,9 @@ def test_generated_schema_same_as_original(
     diff_json_schemas: Callable,
     json_schema_dict: Callable,
 ):
+    # TODO: create instead a fixture that returns a Callable and do these checks
+    # on separate test_* files that follow the same package submodule's hierarchy
+    #
```

Comment:
hmm ok.. will think about it.
I think we need to upgrade the json-schema-diff package also... also I found another one called json-schema-diff-validator... maybe worth a try.